mongodbatlas/data_source_mongodbatlas_event_triggers_test.go

```diff
@@ -0,0 +1,76 @@
+package mongodbatlas
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/mwielbut/pointy"
+	"go.mongodb.org/realm/realm"
+)
+
+func TestAccDataSourceMongoDBAtlasEventTriggers_basic(t *testing.T) {
+	SkipTest(t)
+	var (
+		resourceName = "mongodbatlas_event_trigger.test"
+		projectID    = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+		appID        = os.Getenv("MONGODB_REALM_APP_ID")
+		eventResp    = realm.EventTrigger{}
+	)
+	event := realm.EventTriggerRequest{
+		Name:       acctest.RandomWithPrefix("test-acc"),
+		Type:       "DATABASE",
+		FunctionID: os.Getenv("MONGODB_REALM_FUNCTION_ID"),
+		Disabled:   pointy.Bool(false),
+		Config: &realm.EventTriggerConfig{
+			OperationTypes: []string{"INSERT", "UPDATE"},
```

Comment:
Same as above, maybe a test for a login type and a test for a database type

Reply:
already has in the resource tests including with eventbridge for database, auth and schedule

Reply:
Ah, yup - thanks!